src/main/java/edu/hm/hafner/util/StringContainsUtils.java

```diff
@@ -55,6 +55,37 @@ public static boolean containsAnyIgnoreCase(@Nullable final CharSequence input,
         return false;
     }
 
+    /**
+     * @author Nina Grabow
+     *
+     * Checks if the provided string contains irrespective of case all of the strings in the given array,
+     * handling {@code null} strings. Case-insensitivity is defined as by {@link String#equalsIgnoreCase(String)}.
+     *
+     * @param input
+     *         The string to check, may be {@code null}
+     * @param searchTexts
+     *         The strings to search for, may be empty. Individual CharSequences may be null as well.
+     *
+     * @return {@code true} if all of the search CharSequences are found, {@code false} otherwise.
+     *
+     */
+    public static boolean containsAllIgnoreCase(@Nullable final CharSequence input,
+            @Nullable final String... searchTexts) {
+        if (StringUtils.isEmpty(input)) {
+            return false;
+        }
+        if (searchTexts == null || searchTexts.length == 0) {
+            return false;
+        }
+        boolean result = true;
+        for (String searchText : searchTexts) {
+            if (StringUtils.containsIgnoreCase(input, searchText)) {
+                continue;
+            }
+            result =  false;
+        }
+        return result;
+    }
```

Comment:
Bitte niemals selbst im `master` Branch arbeiten. Immer einen neuen Branch für jedes Feature erstellen, der vom Master-Branch abzweigt. Siehe dazu: [About branches](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-branches)
_Deinen_ Master musst du dann nur noch falls notwendig mit den Änderungen von Herrn Hafners Repository aktualisieren. Siehe dazu: [Syncing a fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork)